picard/tagger.py

```diff
@@ -291,6 +293,24 @@ def __init__(self, picard_args, unparsed_args, localedir, autoupdate):
         if self.autoupdate_enabled:
             self.updatecheckmanager = UpdateCheckManager(parent=self.window)
 
+        from queue import Queue
```

Comment:
We usually move imports at top of the file.

Reply:
Done.